### ISSUE TITLE
Allow activity without ending datetime, to be filled in later

### ISSUE THREAD

**Issue by LostSoulfly** [babybuddy/babybuddy#92]
Allow activity without ending datetime, to be filled in later or allow future times for the end time. This could serve as an estimate and be modified afterwards.

I understand this is essentially what Timers are for, but here's what happens with my wife...

![image](https://user-images.githubusercontent.com/9120519/72675388-06426d00-3a51-11ea-9be4-9e75945d5c7e.png)

... and then she makes a manual entry. Also, it's difficult if not impossible to start a timer 

I'm uncertain how difficult this would be to implement as I have zero JS experience.

**Comment by cdubz** [comment]
Allowing for times in the future should be OK. I'll just need to look at how that may affect some things like graphs...

> Also, it's difficult if not impossible to start a timer

Could you provide more information on what your mean by this? Are you getting some error? Or having trouble with the interface?

**Comment by LostSoulfly** [comment]
Oops, looks like that was an incomplete thought. It was pretty late :)

I think I was trying to say that it's difficult if not impossible to start a timer precisely when performing an action such as feeding or baby falls asleep. Timers work correctly every time, no issues there.

**Comment by cdubz** [comment]
Gotcha. Sounds like issue with the date time widget, too. I have never really loved it and also found it kind of annoying to change preset dates.

#50 #90 #93 all somewhat related. I’ll be working on improving this.

**Comment by LostSoulfly** [comment]
I was thinking, perhaps allow a TBD ending time? It could be exempt from graphs/reports.

**Comment by cdubz** [comment]
Update: the datetime picker widget has been cleaned up (and hopefully improved) in the latest release (`v1.3.0`).

I'm wishy-washy on TBD end times. It would add a good bit of complexity to things and I'm not totally sure it's worth it. I'm interested to hear more about your experience using timers.

A little narrative on how we operate -- 

For feedings we start a timer around when baby starts and stop it when baby is completely done -- this includes, most the time, both breasts, burping, keeping baby awake, etc -- recording the _last_ breast as a the "type" (since that is the most important piece of info to keep track of).

For sleep, we start a timer when baby looks close and just restart that timer as we check on him and inevitably find him gurgling about and staring off in to nothingness. Sometimes we end up restarting the timer for half an hour or more, hah. Ultimately we get close enough to the actual time.

Basically we try not to put much stock in getting anything exactly precise because there isn't necessarily any pay off to the exactness of the data w.r.t what we have been able to get out of it (i.e. a _general_ understanding of what is going on with baby).

Relatedly: we also don't do anything with completed timers, there are hundreds of them in our lists. Someone else actually also recent made an FR to delete them automatically or have a batch option.

**Comment by LostSoulfly** [comment]
I think it's just difficult for my wife and I to wrap our heads around
checking if there's an active timer for whatever activity is currently
happening.  We'll have to attempt timers again.

For feedings I just look at the clock and try to make a mental note of
start and stop times, putting them in manually afterwards. So having a
button for "Now" will be a big help.

I tried to update my docker a few days ago but it didn't appear to work.
I'll look into it again soon as I'm still on 1.2.4.

Thanks!

**Comment by cdubz** [comment]
Oh, I forgot to mention the widget does have a “now” icon now. It’s a little calendar, the default provided by the library.

Re: Docker see #106. There are apparently some issues there that need to be worked out.

**Comment by cdubz** [comment]
Now that Docker issues are cleared up, let me know if you have any issues/feedback on timers use or the changes to the datetime picker widget as you use it.

**Comment by LostSoulfly** [comment]
I think this is a non-issue going forward. The picker widget has been excellent and the only other feedback for timers would be to clean them up after converting them to a feeding/tummy time/etc.
